### PR TITLE
Update OS packages in image

### LINKED ISSUE #4806: Update OS packages in image
### Problem

The container images frequently contains out of date packages from the base image that may contain vulnerabilities. We should keep them up to date even if there's not an updated base image to consume.

### Solution

* Update base image for all mirror node components
* Update postgresql chart and postgres-repmgr image
* Add an extra command to update all OS packages so they at least are up to date as of each tag
```
RUN apt-get update \
    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
    && rm -rf /var/lib/apt/lists/*
```
or
```
RUN apk update && apk upgrade && rm -rf /var/cache/apk/*
```

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by MarcKriguerAtHedera**
**Description**:

* Update base image for all mirror node components
* Update postgresql chart and postgres-repmgr image
* Update all OS packages so they at least are up to date as of each tag
* Change Docker plugins to always pull base image

**Related issue(s)**:

Fixes #4806

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

The various changes were tested locally, but not with unit tests/integration tests.

### CODE REVIEW

File: hedera-mirror-importer/Dockerfile [hunk 1]
```diff
@@ -6,14 +6,18 @@ RUN java -Djarmode=layertools -jar *.jar extract
...
+COPY --chown=1000:1000 --from=builder /app/snapshot-dependencies/ ./
 RUN true  # Workaround https://github.com/moby/moby/issues/37965
-COPY --from=builder /app/spring-boot-loader/ ./
-COPY --from=builder /app/application/ ./
+COPY --chown=1000:1000 --from=builder /app/spring-boot-loader/ ./
+# install OS updates
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+COPY --chown=1000:1000 --from=builder /app/application/ ./
```
Comment (steven-sheehy) [inline hedera-mirror-importer/Dockerfile:20]: apt should be after all copies

File: hedera-mirror-importer/Dockerfile [hunk 2]
```diff
@@ -6,14 +6,18 @@ RUN java -Djarmode=layertools -jar *.jar extract
...
 # Copy artifacts
-COPY --from=builder /app/dependencies/ ./
-COPY --from=builder /app/snapshot-dependencies/ ./
+COPY --chown=1000:1000 --from=builder /app/dependencies/ ./
+COPY --chown=1000:1000 --from=builder /app/snapshot-dependencies/ ./
 RUN true  # Workaround https://github.com/moby/moby/issues/37965
-COPY --from=builder /app/spring-boot-loader/ ./
-COPY --from=builder /app/application/ ./
+COPY --chown=1000:1000 --from=builder /app/spring-boot-loader/ ./
+# install OS updates
```
Comment (steven-sheehy) [inline hedera-mirror-importer/Dockerfile:16]: nit: Move comment to end like others

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: hedera-mirror-importer/Dockerfile (hunk 1 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-importer/Dockerfile]: Done.

File: hedera-mirror-importer/Dockerfile (hunk 2 above)
Comment (MarcKriguerAtHedera) [reply hedera-mirror-importer/Dockerfile]: Done.  (I also noticed that the number of spaces wasn't consistent, so I made it the same in a couple of other `Dockerfile`s. )

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Pull request merged**